Makefile

```diff
@@ -532,7 +532,7 @@ $(CLASS_FILES_BE_JVM_CLASSFILE): $(JAVA_FILES_BE_JVM_CLASSFILE) $(CLASS_FILES_UT
 	$(JAVAC) -cp $(CLASSES_DIR) -d $(CLASSES_DIR) $(JAVA_FILES_BE_JVM_CLASSFILE)
 	touch $@
 
-$(CLASS_FILES_BE_JVM_RUNTIME): $(JAVA_FILES_BE_JVM_RUNTIME) $(CLASS_FILES_UTIL)
+$(CLASS_FILES_BE_JVM_RUNTIME): $(JAVA_FILES_BE_JVM_RUNTIME) $(CLASS_FILES_UTIL) $(CLASS_FILES_BE_INTERPRETER)
```

Comment:
This might be done in a separate clean-up PR: The `dev.flang.be.jvm.runtime` package should contain only the minimum code required to be packaged with a jar created by the JVM backend. So the dependency on the interpreter basically requires all the interpreter to be part of this, which is a bit too much. It would be much better to move the essential code to `dev.flang.be.jvm.runtime` and introduce a dependence from the interpreter backend on the JVM backend's runtime. 
Related to this: I am already a little unhappy about the dependency on `dev.flang.util`, but would not clean this up yet since we will likely need things like `SourceFile` and `SourcePosition` for better error reporting and debugging of code created by the JVM backend.